pennylane_lightning/core/src/simulators/lightning_kokkos/StateVectorKokkos.hpp

```diff
@@ -622,7 +624,9 @@ class StateVectorKokkos final
         case GeneratorOperation::MultiRZ:
             return applyGeneratorMultiRZ(wires, inverse, params);
         default:
+            // LCOV_EXCL_START
             PL_ABORT(std::string("Generator does not exist for ") + opName);
+            // LCOV_EXCL_END
```

Comment:
I wonder why we are excluding it from the coverage checks.

Reply:
It is excluded to increase the % coverage (acting on the denominator).

The `default` case is there just in case (for the future one might say), but it is not covered by the tests first because `applyNamedOperation` is never called directly, second because `gates_indices_[opName]` only registers valid operations and last because `opName` is usually validated before hitting that function.

Reply:
Even when we have a default case, for a just-in-case situation, it is better to test it.
We want to mitigate the number of variables when we are potentially hitting a bug, especially if it is in an unusual situation.